### PR TITLE
fix(chips): Fix chips displaying initial transition on the selection icon. 

### LINKED ISSUE #6183: Chips apply initial transition on the selection icon, onSelectionDone triggered
## Description  
This came up when looking at the failing tests with Ivy https://github.com/IgniteUI/igniteui-angular/issues/6175#issuecomment-555980120 for the igxChip. It appears the selected icon is initially visible and is hidden with a transition, so when you load the Chip demo it shifts a bit. Here's with animations slowed way down:
chip-initial-transition

- The settling into place effect I'm assuming is totally unintended
- There's an issue with onSelectionDone events, since those are hooked on transition events:

1. In the Chip sample in dev demos
2. add `(onSelectionDone)="onChipsSelected('onSelectionDone')"` to all chips in the "Intro to Chips" section
3. Observe the console on page load:
image

### PR DESCRIPTION
**Pull request by skrustev**
Closes #6183

### Additional information (check all that apply):
 - [x] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code
 - [ ] This PR includes API docs for newly added methods/properties
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/chips/chip.component.ts [hunk 1]
```diff
@@ -442,6 +450,11 @@ export class IgxChipComponent extends DisplayDensityBase {
             cancel: false
         };
 
+        // Take the first 2 because there is transition for opacity and width and their order is not certain.
+        fromEvent(this.selectContainer.nativeElement, 'transitionend').pipe(take(2)).subscribe((event) => {
```
Comment (rkaraivanov) [inline projects/igniteui-angular/src/lib/chips/chip.component.ts]: ```typescript .pipe(filter(event => event.propertyName === 'width'), take(1))```
Then you can remove the check for that in the event handler

File: projects/igniteui-angular/src/lib/core/styles/components/chip/_chip-theme.scss [hunk 2]
```diff
@@ -432,11 +432,11 @@
         align-items: center;
         z-index: 1;
         opacity: 1;
-        transition: opacity 120ms $ease-in-out-quad;
+        width: 18px;
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/components/chip/_chip-theme.scss]: This approach breaks the chip in Safari. I don't have the means to currently test it in IE and old Edge.

Comment (ChronosSF) [reply projects/igniteui-angular/src/lib/core/styles/components/chip/_chip-theme.scss]: Please, suggest approach sir :)

**Comment by ChronosSF** [comment]
@simeonoff , please apply the fix as you see fit. We'll verify it for IE11

**Comment by ChronosSF** [comment]
@MayaKirova , please verify this with IE11 and Safari

**Pull request merged**